digitalocean/config.go

```diff
@@ -17,29 +20,57 @@ import (
 )
 
 type Config struct {
-	Token string
+	Token     string
+	AccessID  string
+	SecretKey string
+}
+
+type CombinedConfig struct {
+	client    *godo.Client
+	accessID  string
+	secretKey string
+}
+
+func (c *CombinedConfig) godoClient() *godo.Client { return c.client }
+
+func (c *CombinedConfig) awsClient(region string) (*session.Session, error) {
```

Comment:
Maybe something like s3Client or spacesClient would be a better name?

Reply:
yeah, `spacesClient` is indeed a better and more straightforward name.  I updated the PR.